src/main/java/newbank/server/NewBankClientHandler.java

```diff
@@ -28,7 +28,13 @@ public void run() {
         } else break;
       }
       // The user is authenticated. Get requests from the user and process them.
-      out.println("Log In Successful. What do you want to do?");
+      out.println("Log In Successful");
```

Comment:
You would have to interrogate the server to find out what commands are actually available. Then display the menu.

Kindest Regards,
Jules